pages/404.tsx

```diff
@@ -0,0 +1,23 @@
+import SignDirectionIcon from 'mdi-react/SignDirectionIcon'
+import * as React from 'react'
+import Layout from '../components/Layout'
+
+export class NotFoundPage extends React.Component<any, {}> {
+    public render(): JSX.Element {
+        return (
+            <Layout location={this.props.location} className="bg-white">
+                <div className="container center error-page text-dark">
+                    <div className="error-page__circle rounded-circle">
+                        <div className="error-page__icon">
+                            <SignDirectionIcon />
```

Comment:
Do we know the usage rights around the sign direction icon?

Reply:
This icon comes from [Material Design Icons](https://materialdesignicons.com/) which is claimed to be [free and open source according to this license file](https://github.com/Templarian/MaterialDesign/blob/master/LICENSE). We use [`mdi-react`](https://github.com/levrik/mdi-react) which is a package that contains these icons in the form of React components and that package is MIT.

Reply:
I think we will eventually want a branded image here, is is worthwhile to have a widely used placeholder in the meantime, or can we skip it and add something later?

Reply:
Let us add something later. Tommy could probably give us guidelines